### ISSUE TITLE
Result reproduction

### ISSUE THREAD

**Issue by Xusssssss** [Open3DA/LL3DA#20]
Hello author, I am very interested in your work, but I can't find the corresponding implementation of the results in the paper in the code. May I ask which part of the code was used to obtain the experimental results in the following figure
<img width="423" alt="2c55db5d20086fbfa3aaaaee93ae590" src="https://github.com/Open3DA/LL3DA/assets/116827258/19f710d9-6de7-4a57-988d-83b0cbe56fac">

**Comment by ch3cook-fdu** [comment]
Please follow the following steps:

1. Train the generalist model: `bash scripts/opt-1.3b/train.generalist.sh` or `bash scripts-v0/opt-1.3b/train.generalist.sh`.
2. (optional) Fine-tune for ScanQA: `bash scripts/opt-1.3b/tuning.scanqa.sh`.
3. Inference: `bash scripts/opt-1.3b/eval.scanqa.sh`.

The evaluations of the test set come from the ScanQA benchmark on EvalAI.
